client/reader/stream/index.jsx

```diff
@@ -482,6 +484,10 @@ class ReaderStream extends Component {
 	};
 
 	fetchNextPage = ( options, props = this.props ) => {
+		if ( this.props.disableInfiniteScroll && this.props.items.length > 4 ) {
+			return;
+		}
+
```

Comment:
Some tabs like "First post" need to fetch more than one page to gather more than 1 post, so we check for this.props.items.length > 4 to ensure there are a few posts to display before disabling the infinite scroll.

Reply:
I think we should display at least 10-15 posts before showing signup content.